src/components/Toolbar.js

```diff
@@ -1,6 +1,32 @@
+/* eslint-disable fp/no-let */
 import React from 'react'
 import { connect } from 'react-redux'
 import { shortcutById } from '../shortcuts'
+import {
+  overlayReveal,
+  overlayHide,
+  overlayUpdate
+} from '../action-creators/toolbar'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+
+const isTouchEnabled = () => {
+  return (
+    'ontouchstart' in window ||
+    navigator.maxTouchPoints > 0 ||
+    navigator.msMaxTouchPoints > 0
+  )
+}
+
+(() => {
+  window.addEventListener('mouseup', () => {
+    clearHoldTimer()
+  })
```

Comment:
The extra lambda function is unnecessary here. You can simply do `window.addEventListener('mouseup',  clearHoldTimer)`.

Reply:
Got it.